extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -281,6 +342,59 @@ export class PostgresOverviewPage extends DashboardPage {
 		];
 	}
 
+	private getPodStatus(): PodStatusModel[] {
+		let podModels: PodStatusModel[] = [];
+		const podStatus = this._postgresModel.config?.status.podsStatus;
+
+		podStatus?.forEach(p => {
+			// If a condition of the pod has a status of False, pod is not Ready
+			let status = loc.ready;
+			p.conditions.forEach(c => {
+				if (c.status === 'False') {
+					status = loc.notReady;
+				}
+			});
+
+			const podLabelContainer = this.modelView.modelBuilder.flexContainer().withProps({
+				CSSStyles: { 'alignItems': 'center', 'height': '15px' }
+			}).component();
+
+			const imageComponent = this.modelView.modelBuilder.image().withProps({
+				iconPath: IconPathHelper.postgres,
+				width: iconSize,
+				height: iconSize,
+				iconHeight: '15px',
+				iconWidth: '15px'
+			}).component();
+
+			let podLabel = this.modelView.modelBuilder.text().withProps({
```

Comment:
You could just pass in the string directly instead of making a new component

Reply:
I added it to a flex container so it can be placed together with the postgres image and indented accordingly.